src/Language/PureScript/Publish/ErrorsWarnings.hs

```diff
@@ -234,19 +234,18 @@ spdxExamples =
 
 displayRepositoryError :: RepositoryFieldError -> Box
 displayRepositoryError err = case err of
-  RepositoryFieldMissing ->
+  RepositoryFieldMissing giturl ->
     vcat
       [ para (concat
          [ "The 'repository' field is not present in your package manifest file. "
          , "Without this information, Pursuit would not be able to generate "
-         , "source links in your package's documentation. Please add one - like "
-         , "this, for example:"
```

Comment:
I think I prefer the original text here, actually. We don't want to be too prescriptive: people might have a reason for having `remote.origin.url` differ from what's inside bower.json.